assembl/static2/js/app/components/debate/thread/video.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
Is it possible to have a common component for all phases? Because I suppose that it's a copy/past from the video component in `components/debate/survey/video.jsx`

Reply:
It's a copy paste with modifications that would change the behaviour of the other one.